installation.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+virtualenv -q -p /usr/bin/python3.5 scancode-server
```

Comment:
We are not using Python 3 for this project, but only Python 2.7 for as long as ScanCode proper is nt ported to Python 3
Also, what are -q and -p options? long form are better
Also this is not portable on Windows, is it?
I would much prefer that we adopt the `configure` script from scancode https://github.com/nexB/scancode-toolkit/blob/f8132b344ba3a700907c41038e3d663002bbac63/configure ... Can you look into this?

Reply:
yaa, I'll see to it and see what all things can be changed!!

Reply:
sorry about `-q` and `-p` part, `-q` is for quiet and `-p` is an option to decide the python interpreter to create virtual env which requires python root directory as an argument.

Reply:
Just use the long form in the script please.

Reply:
Done that!!